python/simulators/CMakeLists.txt

```diff
@@ -2,10 +2,17 @@ pybind11_add_module(simulators simulators.cpp SYSTEM)
 
 set_target_properties( simulators PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/python/opm2 )
 
-target_sources(simulators
-  PRIVATE
+if(TARGET flow_libblackoil)
```

Comment:
I am probably missing something, but it seems like the creation of the target is uncondtional. Hence, is the else branch ever evaluated?

Reply:
Well, you are not wrong as such. But I want to keep the option of building it standalone open. There are however other reasons why that does not work currently, so maybe I'm overthinking it.

Reply:
What I mean is that it's currently possible to make a CMakeLists.txt which imports an opmsimulators target from the system, then includes this CMakeLists.txt. That would allow you to build the python bindings separate from opmsimulators lib. But I guess it's not worth it, so I can remove the if'ery.